megamek/src/megamek/common/pathfinder/AeroGroundPathFinder.java

```diff
@@ -59,29 +57,22 @@ protected int getMaximumVelocity(IAero mover) {
     protected IGame game;
     private List<MovePath> aeroGroundPaths;
     protected int maxThrust;
-    private MMLogger logger;
 
     protected AeroGroundPathFinder(IGame game) {
         this.game = game;
-        getLogger().setLogLevel(LOGGER_CATEGORY, LogLevel.DEBUG);
+        MegaMek.getLogger().setLogLevel(LogLevel.DEBUG);
```

Comment:
Same as before, file or system, and why hardcoded.

Reply:
Not mine, can't say.